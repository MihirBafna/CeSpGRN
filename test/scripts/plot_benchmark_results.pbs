#PBS -N plot_benchmark_results
#PBS -q hive-himem
#PBS -l nodes=1:ppn=4
#PBS -l walltime=36:20:00
#PBS -j oe
#PBS -o plot_benchmark_results.out
#PBS -m abe                            
#PBS -M zzhang834@gatech.edu  

cd $PBS_O_WORKDIR
module load anaconda3
conda activate myenv
python plot_benchmark_results.py