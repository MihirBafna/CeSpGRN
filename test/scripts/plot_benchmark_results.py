# In[0]
import pandas as pd
import numpy as np
import torch
import torch.nn
import matplotlib.pyplot as plt
import sys, os
sys.path.append('../../src/')

import matplotlib.pyplot as plt
from scipy.spatial.distance import pdist, squareform

import bmk_beeline as bmk
import warnings
warnings.filterwarnings("ignore")


# TODO: Compare between models, either using the best score or the average score, check if there are significant differences between models


result_dir = "../results/GGM/"
# baseline scores
scores_random = pd.read_csv(result_dir + "score_rand.csv")

# model 1, select the one with the highest median value
alpha = 1
rho = "adaptive"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + ".csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()


# model 2
alpha = 2
rho = "1.7"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + ".csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()

# model 3
alpha = 1
rho = "1.7"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + ".csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()
print()
print("model using no mean")
print()

# model 4
alpha = 1
rho = "adaptive"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + "_nomean.csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()

# model 5
alpha = 2
rho = "1.7"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + "_nomean.csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()

# model 6
alpha = 1
rho = "1.7"
best_nmse = [100, 0, 0]
best_pearson = [0, 0, 0]
best_cos = [0, 0, 0]
for bandwidth in [0.01, 0.1, 1, 10]:
    for lamb in [0.01, 0.1, 1]:
        data = str(bandwidth) + "_" + str(alpha) + "_" + str(lamb) + "_" + rho
        score = pd.read_csv(result_dir + "score_" + data + "_nomean.csv")
        # extract median score across times
        nmse_med = np.median(score["nmse"].values)
        # pos_med = np.median(score["probability of success"].values)
        pearson_med = np.median([eval(x)[0] for x in score["pearson"].values])
        cos_med = np.median(score["cosine similarity"].values)
        print("bandwidth: " + str(bandwidth) + ", alpha: " + str(alpha) + ", lambda: " + str(lamb) + ", rho: " + str(rho))
        print("nmse: " + str(nmse_med) + ", pearson: " + str(pearson_med) + ", cos simi: " + str(cos_med))
        if nmse_med < best_nmse[0]:
            best_nmse[0] = nmse_med
            best_nmse[1] = bandwidth
            best_nmse[2] = lamb
        if pearson_med > best_pearson[0]:
            best_pearson[0] = pearson_med
            best_pearson[1] = bandwidth
            best_pearson[2] = lamb
        if cos_med > best_cos[0]:
            best_cos[0] = cos_med
            best_cos[1] = bandwidth
            best_cos[2] = lamb
print()
print("best nmse: " + str(best_nmse[0]) + ", bandwidth: " + str(best_nmse[1]) + ", lamb: " + str(best_nmse[2]))
print("best pearson: " + str(best_pearson[0]) + ", bandwidth: " + str(best_pearson[1]) + ", lamb: " + str(best_pearson[2]))
print("best cos: " + str(best_cos[0]) + ", bandwidth: " + str(best_cos[1]) + ", lamb: " + str(best_cos[2]))
print()

# %%
