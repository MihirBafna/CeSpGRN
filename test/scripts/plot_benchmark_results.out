---------------------------------------
Begin PBS Prologue Tue Sep 21 13:45:08 EDT 2021
Job ID:     7015811.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   plot_benchmark_results
Queue:      hive-himem
End PBS Prologue Tue Sep 21 13:45:08 EDT 2021
---------------------------------------
bandwidth: 0.01, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 0.937234014333546, pearson: 0.9617950265093597, cos simi: 0.25218426709107766
bandwidth: 0.01, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 0.9496235447331702, pearson: 0.9390437667431176, cos simi: 0.24080107855695831
bandwidth: 0.01, alpha: 1, lambda: 1, rho: adaptive
nmse: 0.9577207199903623, pearson: 0.9169310311247925, cos simi: 0.23102303725135453
bandwidth: 0.1, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 0.9496235447331702, pearson: 0.9390437667431176, cos simi: 0.24095586356039525
bandwidth: 0.1, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 0.9564640447747534, pearson: 0.9267703102844936, cos simi: 0.23684144386246458
bandwidth: 0.1, alpha: 1, lambda: 1, rho: adaptive
nmse: 0.9577261155902184, pearson: 0.9171637199885041, cos simi: 0.23233226000271115
bandwidth: 1, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 0.9564640447747534, pearson: 0.9267703102844936, cos simi: 0.23684144386246458
bandwidth: 1, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 0.9574162641823263, pearson: 0.9192984221968326, cos simi: 0.2352392534534214
bandwidth: 1, alpha: 1, lambda: 1, rho: adaptive
nmse: 0.9578457660650515, pearson: 0.9137040515144044, cos simi: 0.23301249209939565
bandwidth: 10, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 0.9574162641823263, pearson: 0.9192984221968326, cos simi: 0.23482416164724207
bandwidth: 10, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 0.9577867047813392, pearson: 0.913709151198578, cos simi: 0.2350898410947908
bandwidth: 10, alpha: 1, lambda: 1, rho: adaptive
nmse: 0.958053648861299, pearson: 0.910544928416346, cos simi: 0.23387572741924134

best nmse: 0.937234014333546, bandwidth: 0.01, lamb: 0.01
best pearson: 0.9617950265093597, bandwidth: 0.01, lamb: 0.01
best cos: 0.25218426709107766, bandwidth: 0.01, lamb: 0.01

bandwidth: 0.01, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 0.0400048961275328, pearson: 0.9807953834191749, cos simi: 0.9817364706429865
bandwidth: 0.01, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 0.1716347687860551, pearson: 0.9562595584874726, cos simi: 0.958462722574546
bandwidth: 0.01, alpha: 2, lambda: 1, rho: 1.7
nmse: 0.29782255495457743, pearson: 0.9365683829313942, cos simi: 0.9397201165955704
bandwidth: 0.1, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 0.1716347687860551, pearson: 0.9562595584874726, cos simi: 0.958462722574546
bandwidth: 0.1, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 0.2943407534536709, pearson: 0.9450424064605737, cos simi: 0.9477407676885656
bandwidth: 0.1, alpha: 2, lambda: 1, rho: 1.7
nmse: 0.3007723656842549, pearson: 0.9360845741458491, cos simi: 0.9393610928916543
bandwidth: 1, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 0.2943407534536709, pearson: 0.9450424064605737, cos simi: 0.9477407676885656
bandwidth: 1, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 0.3007723656842549, pearson: 0.9389066202105223, cos simi: 0.9419878380773332
bandwidth: 1, alpha: 2, lambda: 1, rho: 1.7
nmse: 0.30810188097534075, pearson: 0.932617336638558, cos simi: 0.9359226389972761
bandwidth: 10, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 0.3007723656842549, pearson: 0.9387456256993806, cos simi: 0.9417716527347055
bandwidth: 10, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 0.30810188097534075, pearson: 0.932617336638558, cos simi: 0.9359226389972761
bandwidth: 10, alpha: 2, lambda: 1, rho: 1.7
nmse: 0.313515523894043, pearson: 0.9299849915577678, cos simi: 0.9334342507134483

best nmse: 0.0400048961275328, bandwidth: 0.01, lamb: 0.01
best pearson: 0.9807953834191749, bandwidth: 0.01, lamb: 0.01
best cos: 0.9817364706429865, bandwidth: 0.01, lamb: 0.01

bandwidth: 0.01, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 0.04000515578392975, pearson: 0.9807953475504885, cos simi: 0.9817364223578362
bandwidth: 0.01, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 0.1716347954651075, pearson: 0.9562596478964227, cos simi: 0.9584627865409703
bandwidth: 0.01, alpha: 1, lambda: 1, rho: 1.7
nmse: 0.2978230231024588, pearson: 0.9365684722026277, cos simi: 0.9397201936160848
bandwidth: 0.1, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 0.1716347954651075, pearson: 0.9562596478964227, cos simi: 0.9584627865409703
bandwidth: 0.1, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 0.29434102386428135, pearson: 0.9450424881987693, cos simi: 0.947740901693381
bandwidth: 0.1, alpha: 1, lambda: 1, rho: 1.7
nmse: 0.3007727200868253, pearson: 0.9360846707852531, cos simi: 0.9393611758249945
bandwidth: 1, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 0.29434102386428135, pearson: 0.9450424881987693, cos simi: 0.947740901693381
bandwidth: 1, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 0.3007727200868253, pearson: 0.9389067199927932, cos simi: 0.9419879025573408
bandwidth: 1, alpha: 1, lambda: 1, rho: 1.7
nmse: 0.3081023207086185, pearson: 0.9326174968531729, cos simi: 0.9359228036692604
bandwidth: 10, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 0.3007727200868253, pearson: 0.9387457391255005, cos simi: 0.9417717438929565
bandwidth: 10, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 0.3081023207086185, pearson: 0.9326174968531729, cos simi: 0.9359228036692604
bandwidth: 10, alpha: 1, lambda: 1, rho: 1.7
nmse: 0.3135160758071627, pearson: 0.9299851109859544, cos simi: 0.9334343733980064

best nmse: 0.04000515578392975, bandwidth: 0.01, lamb: 0.01
best pearson: 0.9807953475504885, bandwidth: 0.01, lamb: 0.01
best cos: 0.9817364223578362, bandwidth: 0.01, lamb: 0.01


model using no mean

bandwidth: 0.01, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 50.408994133540105, pearson: -0.03621281024291742, cos simi: -0.034198406899976955
bandwidth: 0.01, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 31.59454239083102, pearson: -0.023107617427440115, cos simi: -0.01936768369222725
bandwidth: 0.01, alpha: 1, lambda: 1, rho: adaptive
nmse: 16.69038392579521, pearson: -0.01618166064490417, cos simi: -0.005245366497066049
bandwidth: 0.1, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 28.09663156279904, pearson: -0.017553020846010996, cos simi: -0.00998432263291075
bandwidth: 0.1, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 21.78699826939164, pearson: -0.01677145293823139, cos simi: -0.0084436274257036
bandwidth: 0.1, alpha: 1, lambda: 1, rho: adaptive
nmse: 16.437715892957662, pearson: -0.01437901927317508, cos simi: -0.0034690292195704003
bandwidth: 1, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 20.891750581320785, pearson: -0.015091488585434645, cos simi: -0.0055403505406321
bandwidth: 1, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 18.929965021921753, pearson: -0.01345026934016635, cos simi: -0.0059257050747441
bandwidth: 1, alpha: 1, lambda: 1, rho: adaptive
nmse: 16.12151186580667, pearson: -0.011511297020923943, cos simi: -0.0020400301838693
bandwidth: 10, alpha: 1, lambda: 0.01, rho: adaptive
nmse: 18.66971836070047, pearson: -0.012078633693713137, cos simi: -0.002836600799748
bandwidth: 10, alpha: 1, lambda: 0.1, rho: adaptive
nmse: 17.725896544486012, pearson: -0.010465436196153489, cos simi: -0.00221548852152555
bandwidth: 10, alpha: 1, lambda: 1, rho: adaptive
nmse: 15.6768686662919, pearson: -0.009108504174745423, cos simi: 0.0

best nmse: 15.6768686662919, bandwidth: 10, lamb: 1
best pearson: 0, bandwidth: 0, lamb: 0
best cos: 0, bandwidth: 0, lamb: 0

bandwidth: 0.01, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 198.0287939659787, pearson: -0.036283569412370896, cos simi: -0.034205610602748154
bandwidth: 0.01, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 123.19148592689959, pearson: -0.02318439259785044, cos simi: -0.01939610141054625
bandwidth: 0.01, alpha: 2, lambda: 1, rho: 1.7
nmse: 63.60491147940696, pearson: -0.016188905487444376, cos simi: -0.005055427706264701
bandwidth: 0.1, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 109.17508105693605, pearson: -0.017525848177083355, cos simi: -0.009989872663512
bandwidth: 0.1, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 83.96642778048276, pearson: -0.016770993587468468, cos simi: -0.00844278895879955
bandwidth: 0.1, alpha: 2, lambda: 1, rho: 1.7
nmse: 62.736266123559936, pearson: -0.014403299175658971, cos simi: -0.00334243530329075
bandwidth: 1, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 80.46117595964886, pearson: -0.015163318989605157, cos simi: -0.0055022374387219
bandwidth: 1, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 72.58481816467172, pearson: -0.013730157398109191, cos simi: -0.005949907858609899
bandwidth: 1, alpha: 2, lambda: 1, rho: 1.7
nmse: 61.46335513800062, pearson: -0.011783158520872514, cos simi: -0.0019654629419521502
bandwidth: 10, alpha: 2, lambda: 0.01, rho: 1.7
nmse: 71.4501239675059, pearson: -0.012386311343752909, cos simi: -0.0027808975751261
bandwidth: 10, alpha: 2, lambda: 0.1, rho: 1.7
nmse: 67.92568833474954, pearson: -0.010596211026294213, cos simi: -0.00219880599772155
bandwidth: 10, alpha: 2, lambda: 1, rho: 1.7
nmse: 59.60420889668906, pearson: -0.009220140396119392, cos simi: 0.0

best nmse: 59.60420889668906, bandwidth: 10, lamb: 1
best pearson: 0, bandwidth: 0, lamb: 0
best cos: 0, bandwidth: 0, lamb: 0

bandwidth: 0.01, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 198.02790710791442, pearson: -0.03628358771930504, cos simi: -0.0342056300994559
bandwidth: 0.01, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 123.19089547582814, pearson: -0.023184361934693802, cos simi: -0.019396081054345753
bandwidth: 0.01, alpha: 1, lambda: 1, rho: 1.7
nmse: 63.6047324769882, pearson: -0.016188921656193952, cos simi: -0.00505541739400625
bandwidth: 0.1, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 109.17475824983937, pearson: -0.017525864270641754, cos simi: -0.0099899356004185
bandwidth: 0.1, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 83.96625249341479, pearson: -0.016770993594243038, cos simi: -0.00844279606555815
bandwidth: 0.1, alpha: 1, lambda: 1, rho: 1.7
nmse: 62.73614984152549, pearson: -0.014403340710386573, cos simi: -0.00334237529835545
bandwidth: 1, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 80.46091635210965, pearson: -0.015163409177296578, cos simi: -0.0055021370978279496
bandwidth: 1, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 72.58460808988934, pearson: -0.013730165381315553, cos simi: -0.0059499095665937005
bandwidth: 1, alpha: 1, lambda: 1, rho: 1.7
nmse: 61.46319077519337, pearson: -0.0117832080714891, cos simi: -0.0019655310409477
bandwidth: 10, alpha: 1, lambda: 0.01, rho: 1.7
nmse: 71.44996947046639, pearson: -0.012386324799532517, cos simi: -0.00278089953513945
bandwidth: 10, alpha: 1, lambda: 0.1, rho: 1.7
nmse: 67.92549644843922, pearson: -0.010596217245736035, cos simi: -0.00219884634337825
bandwidth: 10, alpha: 1, lambda: 1, rho: 1.7
nmse: 59.604062814312414, pearson: -0.009220129468161269, cos simi: 0.0

best nmse: 59.604062814312414, bandwidth: 10, lamb: 1
best pearson: 0, bandwidth: 0, lamb: 0
best cos: 0, bandwidth: 0, lamb: 0

---------------------------------------
Begin PBS Epilogue Tue Sep 21 13:45:28 EDT 2021
Job ID:     7015811.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   plot_benchmark_results
Resources:  nodes=1:ppn=4,walltime=36:20:00,neednodes=1:ppn=4
Rsrc Used:  cput=00:00:17,vmem=0kb,walltime=00:00:20,mem=0kb,energy_used=0
Queue:      hive-himem
Nodes:     
atl1-1-01-018-38.pace.gatech.edu
End PBS Epilogue Tue Sep 21 13:45:28 EDT 2021
---------------------------------------
